events.md

```diff
@@ -11,13 +11,17 @@ photos:
 
 {%- assign today = site.time | date: '%Y-%m-%d' -%}
 ## Upcoming Events
```

Comment:
Maybe we just remove the idea of Past and Upcoming events and have a big table with all events with the most recent ones on the top

Reply:
Oh, okay...